odoo/cli/neutralize.py

```diff
@@ -20,28 +18,29 @@ class Neutralize(Command):
     def run(self, args):
         parser = odoo.tools.config.parser
         group = optparse.OptionGroup(parser, "Neutralize", "Neutralize the database specified by the `-d` argument.")
+        group.add_option("--stdout", action="store_true", dest="output_code", help="Output the neutralization SQL instead of applying it")
         parser.add_option_group(group)
-        odoo.tools.config.parse_config(args)
+        opt = odoo.tools.config.parse_config(args)
 
         dbname = odoo.tools.config['db_name']
         if not dbname:
             _logger.error('Neutralize command needs a database name. Use "-d" argument')
             sys.exit(1)
 
-        registry = odoo.registry(dbname)
-        _logger.info('Starting database neutralization')
+        if not opt.output_code:
+            _logger.info('Starting %s database neutralization', dbname)
+
         try:
-            with registry.cursor() as cr:
-                env = odoo.api.Environment(cr, odoo.SUPERUSER_ID, {})
-                for model in env.values():
-                    model._neutralize()
-                env['ir.config_parameter'].set_param('database.is_neutralized', True)
-                # Signal changes so the config parameter we just set gets updated in a
-                # running database without requiring a restart.
-                env.registry.signal_changes()
-                with open(Path(__file__).parent / 'neutralize_watermarks.xml', 'rb') as f:
-                    odoo.tools.convert_xml_import(cr, "__neutralize__", f)
+            with odoo.sql_db.db_connect(dbname).cursor() as cursor:
+                installed_modules = odoo.modules.neutralize.get_installed_modules(cursor)
+                code = odoo.modules.neutralize.get_neutralization_sql(installed_modules)
+                if opt.output_code:
```

Comment:
The attribute `output_code` is misleading IMO.  It's not "code" (which immediately suggests Python code), it's some SQL, or "SQL script".  Let me suggest a name that says exactly what it means
```suggestion
                if opt.to_stdout:
```